CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_s8.c

```diff
@@ -40,46 +40,109 @@
  * @{
  */
 
```

Comment:
Please update the version of the file.. increment the major revision here..The date and the copyright year.. You can have a look at any recently merged PR for reference.